scaleway/resource_lb_ip.go

```diff
@@ -33,11 +33,17 @@ func resourceScalewayLbIP() *schema.Resource {
 				Computed:    true,
 				Description: "The reverse domain name for this IP",
 			},
-			"region":          regionComputedSchema(),
-			"zone":            zoneSchema(),
+			"zone": {
+				Type:             schema.TypeString,
+				Description:      "The zone you want to attach the resource to",
+				Optional:         true,
+				ForceNew:         true,
+				ValidateDiagFunc: validateStringInSliceWithWarning(AllZones(), "zone"),
+				Default:          scw.ZoneFrPar1,
```

Comment:
It should be the zone of the provider.

Reply:
Yes, well seen. we should set zoneSchema back.